src/components/Frame/Frame.tsx

```diff
@@ -66,340 +56,292 @@ const APP_FRAME_NAV = 'AppFrameNav';
+  const skipToMainContentTarget = skipToContentTarget ? null : (
+    // eslint-disable-next-line jsx-a11y/anchor-is-valid
+    <a
+      id={APP_FRAME_MAIN_ANCHOR_TARGET}
+      ref={skipToMainContentTargetNode}
+      tabIndex={-1}
+    />
+  );
+
+  return (
+    <FrameContext.Provider value={context}>
+      <div
+        className={frameClassName}
+        {...layer.props}
+        {...navigationAttributes}
+      >
+        {skipMarkup}
+        {topBarMarkup}
+        {navigationMarkup}
+        {contextualSaveBarMarkup}
+        {loadingMarkup}
+        {navigationOverlayMarkup}
+        <main
+          className={styles.Main}
+          id={APP_FRAME_MAIN}
+          data-has-global-ribbon={Boolean(globalRibbon)}
         >
-          {intl.translate('Polaris.Frame.skipToContent')}
-        </a>
+          {skipToMainContentTarget}
+          <div className={styles.Content}>{children}</div>
+        </main>
+        <ToastManager toastMessages={toastMessages} />
+        {globalRibbonMarkup}
       </div>
-    );
-
-    const navigationAttributes = navigation
-      ? {
-          'data-has-navigation': true,
-        }
-      : {};
-
-    const frameClassName = classNames(
-      styles.Frame,
-      navigation && styles.hasNav,
-      topBar && styles.hasTopBar,
-    );
-
-    const navigationOverlayMarkup =
-      showMobileNavigation && isNavigationCollapsed ? (
-        <Backdrop
-          belowNavigation
-          onClick={this.handleNavigationDismiss}
-          onTouchStart={this.handleNavigationDismiss}
-        />
-      ) : null;
-
-    const skipToMainContentTarget = skipToContentTarget ? null : (
-      // eslint-disable-next-line jsx-a11y/anchor-is-valid
-      <a
-        id={APP_FRAME_MAIN_ANCHOR_TARGET}
-        ref={this.skipToMainContentTargetNode}
-        tabIndex={-1}
-      />
-    );
-
-    const context = {
-      showToast: this.showToast,
-      hideToast: this.hideToast,
-      startLoading: this.startLoading,
-      stopLoading: this.stopLoading,
-      setContextualSaveBar: this.setContextualSaveBar,
-      removeContextualSaveBar: this.removeContextualSaveBar,
-    };
-
-    return (
-      <FrameContext.Provider value={context}>
-        <div
-          className={frameClassName}
-          {...layer.props}
-          {...navigationAttributes}
-        >
-          {skipMarkup}
-          {topBarMarkup}
-          {navigationMarkup}
-          {contextualSaveBarMarkup}
-          {loadingMarkup}
-          {navigationOverlayMarkup}
-          <main
-            className={styles.Main}
-            id={APP_FRAME_MAIN}
-            data-has-global-ribbon={Boolean(globalRibbon)}
-          >
-            {skipToMainContentTarget}
-            <div className={styles.Content}>{children}</div>
-          </main>
-          <ToastManager toastMessages={toastMessages} />
-          {globalRibbonMarkup}
-          <EventListener event="resize" handler={this.handleResize} />
```

Comment:
How are we handling this now? I can't spot an equivalent definition to catch window resizing

Reply:
I thought `useMediaQuery` does trigger `resize` for us here.

Reply:
I'm not familiar with how useMediaQuery does its thing. @AndrewMusgrave can you confirm?

Reply:
I've added resize back, nice catch!

Reply:
I'm not sure how often `globalRibbonHeight` should recalculated, I've added `resize` back for now, but `useMediaQuery` might be used as well